docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Troubleshooting"
+weight: 8
+icon: "faq"
+---
+
+## Reporting a problem
+
+Before you report a problem, make sure to retrieve the necessary information from your cluster.
+
+### Automatic log & state collection
+
+You collect information in a Kubernetes cluster using the Cilium CLI:
+
+```shell-session
+$ cilium-cli sysdump
```

Comment:
```suggestion
cilium-cli sysdump
```
[we don't include the command](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt) prompt for faster copy/paste. I know this is a debate because it also brings information.

Reply:
ah, sorry. I keep forgetting.